deployment/helm/skaha/skaha-config/launch-carta.yaml

```diff
@@ -31,6 +31,13 @@ spec:
         supplementalGroups: [${skaha.supgroups}]
       priorityClassName: uber-user-preempt-medium
       hostname: "${software.hostname}"
+      initContainers:
+      - name: setup-access-token
+        image: busybox
+        command: ["sh", "-c", "echo '${access_token}' > /etc/tokens/access-token"]
```

Comment:
@andamian - Could you please advise here?  Where do/should client tools look for tokens to be present on a machine?  Something like ~/.ssl/cadcproxy.pem, but for tokens, and more general and standard for the python community.

Reply:
I'm still a newbie when it comes to the use of tokens. That being said, the tokens must be stored in user space to avoid collision with other users. I don't know of any "standard" cache directory (such as `~/.ssl/`). Maybe `~/.tokens`?

Does IAM support refresh tokens? Access tokens are typically short lived tokens that might be stored in memory for the duration of the cli application whereas refresh tokens are longer lived tokens that are used to retrieve access tokens and can be stored locally in a file.